code/game/mecha/mecha.dm

```diff
@@ -843,6 +843,40 @@
 		. = ..(obstacle)
 	return
 
+
+/obj/mecha/proc/trample(var/mob/living/H)
+	if(!isliving(H))
+		return
+
+	var/obj/item/mecha_parts/component/actuator/actuator = internal_components[MECH_ACTUATOR]

```

Comment:
This should probably check if the exosuit is phasing, since they're not entirely corporeal.